speedy-recovery-client/src/service/FhirDataMappingService.js

```diff
@@ -63,6 +64,7 @@ const fhirMapAppointment = (resource, version) => {
   if (version) {
     switch (version[0]) {
       case "1":
+        return null;
```

Comment:
Please revert all these `case "1"` changes, because the FHIR DSTU2 standard, somewhat misleadingly is actually the version number `1.0.2`. Please make sure to adjust the test cases as well.

Reply:
Also, please add that to the documentation at the top of the class, it's important and not evident